src/main/webapp/components/sources-select/sources-select.js

```diff
@@ -1,17 +1,19 @@
-import React, { useEffect } from 'react'
```

Comment:
not sure why the diff came out like this, but this file was mostly copied from the source select in user settings